### PR TITLE
new option --curl to dump request as curl

### PR DESCRIPTION
**Pull request by hzakher**
using http2curl to dumpRequest() makes it easier to use the generated curl command in debugging/testing

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,6 +7,9 @@ require (
 	github.com/mattn/go-colorable v0.0.9 // indirect
 	github.com/mattn/go-isatty v0.0.4 // indirect
 	github.com/pkg/errors v0.8.1
+	github.com/smartystreets/goconvey v1.6.4 // indirect
```
Comment (dpfg) [inline go.mod:10]: Suspicious dependency. Seems like http2curl doesn't have this as a dependency. Could you elaborate what is the purpose of this?

File: main.go [hunk 2]
```diff
@@ -81,6 +85,7 @@ func main() {
 
 	flag.BoolVar(&infoFlag, "i", false, "Enable info mode. Print request and response details.")
 	flag.BoolVar(&infoFlag, "info", false, "Enable info mode. Print request and response details.")
+	flag.BoolVar(&curlFlag, "curl", false, "Enable info mode. Print request and response details. Request is printed as curl")
```
Comment (dpfg) [inline main.go:88]: `--info-curl` would make more sense in this case

**Comment by dpfg** [comment]
@hzakher thank you for creating PR. Could you provide a bit more context around use cases here - https://github.com/kajf/bozr/issues/108?

**Comment by hzakher** [comment]
thanks for your feedback :)
i adjusted the PR as suggested. i'll close this one

**Pull request closed without merging**